### PR TITLE
Use the api client instead of retriever classes

### PR DESCRIPTION
**Pull request by agrare**
Most of the Retriever classes were just wrappers around simple calls to the api client.  This can be greatly simplified by just calling the api client directly.

### CODE REVIEW

**Comment by agrare** [comment]
cc @carbonin

File: lib/topological_inventory/openshift/operations/worker.rb [hunk 1]
```diff
@@ -40,33 +40,34 @@ def stop
...
+          order_service(msg.payload)
         rescue => e
           logger.error(e.message)
           logger.error(e.backtrace.join("\n"))
           nil
         end
 
-        def order_service(service_plan_id, order_params)
+        def order_service(payload)
+          service_plan_id = payload["service_plan_id"].to_s
```
Comment (carbonin) [inline lib/topological_inventory/openshift/operations/worker.rb]: This wasn't calling `.to_s` before, right? Will adding it cause any problems?

Comment (agrare) [reply lib/topological_inventory/openshift/operations/worker.rb]: It was in the retriever https://github.com/ManageIQ/topological_inventory-collector-openshift/pull/41/files#diff-ee6c74bbf8bf14cfc5756b3f280d5e66L9

**Comment by agrare** [comment]
> Can you also kill the base retriever class as the only remaining subclass is the AuthenticationRetriever which doesn't use the initialized client object?

:+1: yes

**Comment by agrare** [comment]
@carbonin done

**Pull request merged**